tuxemon/states/combat/combat.py

```diff
@@ -423,6 +425,7 @@ def transition_phase(self, phase: CombatPhase) -> None:
             self.players[0].set_party_status()
             var = self.players[0].game_variables
             var["battle_last_result"] = "ran"
+            var["run_attempts"] = +1
```

Comment:
Either += 1 or = 1

Not sure what you're trying to do here

Reply:
+= 1! Thanks

Reply:
> Attempts should increase by +1 after success is checked, not before (i.e. the first attempt should use 0 rather than 1)
Each successful attempt increase the number (+1), as requested.